src/components/ComposedModal/ComposedModal.js

```diff
@@ -174,38 +231,58 @@ export class ModalHeader extends Component {
   }
 }
 
-export class ModalBody extends Component {
```

Comment:
Seems that you are cleaning up stuffs a bit, but I couldn’t find where `<ModalBody>` is gone - Any pointers?

Reply:
Haha yeah, totally missed this 😅 adding back in!